src/common/utils.js

```diff
@@ -25,13 +25,47 @@ import RequestTemplates from 'features/RequestForm/RequestTemplates'
 
 const { Item, SubMenu } = Menu
 
+export const getDataLocalStorage = type =>
+  JSON.parse(window.localStorage.getItem('currentUser'))[type]
```

Comment:
If localStorage have not `currentUser` then you get uncaught exception like `Cannot read property type of undefined`. Fix that in advance